extension/lib/main.js

```diff
@@ -46,7 +46,8 @@ exports.main = function (options, callbacks) {
     height: 107,
```

Comment:
Whatever works best. Feel free to merge and fix the remaining comments here, yeah.